robotbase/src/main/java/com/team2357/frc2022/subsystems/FeederSubsystem.java

```diff
@@ -1,21 +1,20 @@
 package com.team2357.frc2022.subsystems;
 
 import com.team2357.frc2022.Constants;
+import com.team2357.frc2022.arduino.ArduinoSensorState;
 import com.team2357.lib.arduino.ArduinoUSBController;
 import com.team2357.lib.subsystems.ClosedLoopSubsystem;
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
 public class FeederSubsystem extends ClosedLoopSubsystem {
     private WPI_TalonSRX m_feederMotor;
-    private ArduinoUSBController m_arduinoIRSensor;
+    private ArduinoSensorState m_isCargoAtFeederWheel;
```

Comment:
The variable here isn't really a "getter" function itself. I'd probably name it something different like "m_feederSensor" so when you call it below it's `m_cargoFeeder.getState()` which makes more sense to me.